src/components/Navbar.jsx

```diff
@@ -0,0 +1,84 @@
+import { NavLink, useNavigate } from "react-router-dom";
+import { useAuthContext } from "@/context/AuthContext";
+import React from "react";
+
+const links = [
+    { path: '/', text: 'Home' },
+    { path: 'about', text: 'About' },
+    { path: 'profile', text: 'Profile' },
+    { path: 'login', text: 'Login' },
+  ];
+
+const Navbar = () => {
+
+    const navigate = useNavigate()
+    const { user, logout } = useAuthContext()
+    
+    const handleLogout = () => {
+        logout()
+        navigate('/login')
+    }
+    
+    
+    return ( 
+    <>
+        <nav className="navbar">
+            {/* <ul>
+                {links.map((link)=>{
+                    return (
+                        <li key={link.text}>
+                            <NavLink 
+                            to={link.path}
+                            // className={({isActive})=>
+                            // isActive ? 'active__class' : undefined
+                            // }
+                            >{link.text}
+                            </NavLink>
+                        </li>
+                    )
+                })}
+            </ul> */}
```

Comment:
Kindly remember to remove commented/unused code. This makes the app lighter. 😄

Reply:
Thank you so much @Bennyjoez